portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro

```diff
@@ -468,6 +468,16 @@ definition {
 			expected = "${expectedTotalElement}");
 	}
 
+	macro assertStructuredContentIdIsExpected {
+		Variables.assertDefined(parameterList = "${responseToParse},${structuredContentId}");
+
+		var structuredContentIdexpected = JSONUtil.getWithJSONPath("${responseToParse}", "$..['items'][?(@.creator)].id");
```

Comment:
Before the assert used the title of the web content, then it was changed so that this assert is based on the ID.

Reply:
are you checking here the ID of a creator?

Reply:
Here I am comparing the IDs that were obtained with the expected with those that returned after the request. I didn't understand the question well 🤔🤔🤔